### PR TITLE
Ohm Tswap

### PR DESCRIPTION
**Pull request by Joeysantoro**

### CODE REVIEW

File: proposals/description/fip_79.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      values: '0',
+      method: 'allocateTribe(address,uint256)',
+      arguments: ['{feiDAOTimelock}', '3747957360000000000000000'],
+      description: 'Send 3.74M TRIBE to Fei DAO Timelock'
+    },
+    {
+      target: 'tribe',
+      values: '0',
+      method: 'approve(address,uint256)',
+      arguments: ['{ohmEscrow}', '3747957360000000000000000'],
```
Comment (ElliotFriedman) [inline proposals/description/fip_79.ts:17]: I'm not seeing `ohmEscrow` anywhere in mainnetAddresses.ts

Comment (Joeysantoro) [reply proposals/description/fip_79.ts:17]: its deployed and exported in the FIP-79 script. Will be added once the initial review is complete

**Review by ElliotFriedman — APPROVED** [review]
LGTM!

**Comment by ElliotFriedman** [comment]
Looked at all of the addresses, double checked both numbers and it looks good!

**Pull request merged**